.rubocop.yml

```diff
@@ -30,3 +30,6 @@ Rails/InverseOf:
 Lint/MissingSuper:
   Enabled: false
 
+Metrics/ClassLength:
+  Exclude:
+    - app/jobs/update_from_saml_service.rb
```

Comment:
In the AAF code base, I've generally seen per-class exclusions done inline, for from `.rubocop.yml`

Reply:
The `.rubocop.yml` file needs rebasing anyway, I'll change this in the rebase.